advanced_html_css/animation/transitions.md

```diff
@@ -108,6 +108,7 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
+- For a deeper dive on stacking context, check out this [MDN article](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context).
```

Comment:
"MDN article" alone is not descriptive enough as link text - the below will be more accessible. 

```suggestion
- For a deeper dive, check out this [MDN article on the stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context).
```